.circleci/config.yml

```diff
@@ -18,35 +18,35 @@ jobs:
 
   staging-test:
     docker:
-      - image: msheiny/pressfreedomci:latest
+      - image: quay.io/freedomofpress/circleci-docker:latest
```

Comment:
:+1:  Where are the source files for these images? Let's post Dockerfiles to https://github.com/freedomofpress/containers

Reply:
✔️ copied over to that repo in `master`